tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand.cfg

```diff
@@ -28,11 +28,15 @@
 # Copyright (c) 2012 by Delphix. All rights reserved.
 #
 
+#
+# Copyright (c) 2017 Lawrence Livermore National Security, LLC.
+#
+
 
-export org_size=1g
-export exp_size=2g
+export org_size=100m
+export exp_size=200m
```

Comment:
Good point.  When running the test case without this change I got complaints about running out of space, but now I'm not sure why.  I'll look into it more, maybe this part of the change is unnecessary.